android/src/test/java/apitests/java/CommonApiTests.java

```diff
@@ -0,0 +1,149 @@
+package apitests.java;
+
+import android.app.Activity;
+import android.content.Context;
+
+import com.revenuecat.purchases.DangerousSettings;
+import com.revenuecat.purchases.LogHandler;
+import com.revenuecat.purchases.Store;
+import com.revenuecat.purchases.common.PlatformInfo;
+import com.revenuecat.purchases.hybridcommon.CommonKt;
+import com.revenuecat.purchases.hybridcommon.ErrorContainer;
+import com.revenuecat.purchases.hybridcommon.OnResult;
+import com.revenuecat.purchases.hybridcommon.OnResultAny;
+import com.revenuecat.purchases.hybridcommon.OnResultList;
+
+import java.util.List;
+import java.util.Map;
+
+@SuppressWarnings({"unused", "deprecation"})
+class CommonApiTests {
+    private void checkCheckSetAllowSharingAppStoreAccount(Boolean enabled) {
```

Comment:
Why not `boolean`?

Reply:
Well, the compiler won't really help in this case, since the conversion from Kotlin's `Boolean` doesn't make the java side not nullable. But it's true that strictly speaking, this can be `boolean` since it shouldn't ever be null, so I will make the change everywhere except where we have nullable booleans (1 place only). Thanks!